.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+  "singleQuote": false
```

Comment:
This is the default value, so lets get rid of this file

Reply:
I know but I have a setting in my editor to apply prettier only with a config file :D
I can change it for sure but unless adding the file will cause issues let's keep it

Reply:
for example, I had different leftover settings from my previous company, with this file explicitly setting it we can be more sure

Reply:
please change your editor settings then. Its an unnecessary file that has default values in it. It adds confusion (people will look at it and wonder why it exists, and which features we have customized).

In general we should avoid adding files to support a single persons development environment to a repo.